src/greyVest/Popover.js

```diff
@@ -1,32 +1,44 @@
 import React from 'react'
 import _ from 'lodash/fp'
-import OutsideClickHandler from 'react-outside-click-handler'
-import { observer } from 'mobx-react'
+import F from 'futil'
+import Popup from 'reactjs-popup'
 import { openBinding } from './utils'
 import { expandProp } from '../utils/react'
 
-// Simple popover
-let Popover = ({ isOpen, onClose, children, style, contained }) =>
-  isOpen && (
-    <OutsideClickHandler onOutsideClick={_.debounce(0, onClose)}>
-      <div style={{ position: contained ? '' : 'relative' }}>
-        <div
-          className="popover"
-          style={{
-            position: 'absolute',
-            Index: 100,
-            fontWeight: 'normal',
-            textAlign: 'left',
-            background: 'white',
-            border: '1px solid #ebebeb',
-            zIndex: 20,
-            ...style,
-          }}
-        >
-          {children}
-        </div>
-      </div>
-    </OutsideClickHandler>
-  )
+let Popover = ({
+  // eslint-disable-next-line no-unused-vars
+  open, // lens
+  isOpen,
+  style,
+  trigger,
+  arrow,
+  position,
+  children,
+  closeOnPopoverClick = true,
+  ...props
+}) => (
+  <Popup
+    open={open && isOpen} // external state if lens is passed
```

Comment:
`open` is expanded into `isOpen` and `onClose`: https://github.com/smartprocure/contexture-react/blob/f9ceefe1cd1ad5a8c16e245043a460ed8e654590/src/greyVest/utils.js#L3 so we can just use `isOpen` here.

Also, according to the docs if you use `open` you need to pass `onClose` as well (https://react-popup.elazizi.com/controlled-popup).

Here's the API of our current popover:

1. `<Popover open={lens} />`
2. `<Popover isOpen={bool} onClose={fn} />`

We need to support that + `trigger` so

- If `isOpen` is not undefined, pass `open={isOpen}` and `onClose={onClose}` to `Popup`
- if `trigger` is not undefined, don't pass `open` or `onClose`

Reply:
`onClose` is getting passed to the Popup in the `...props`

Reply:
Changed to using `onClose` explicitly and added comments